block/submit.go

```diff
@@ -93,9 +102,6 @@ func (m *Manager) submitNextBatch(ctx context.Context) (uint64, error) {
 	}
 
 	// Submit batch to SL
-	// TODO(omritoptix): Handle a case where the SL submission fails due to syncTarget out of sync with the latestHeight in the SL.
```

Comment:
removed this todo, as we're in `aggregator` mode, and expect our state to be source of truth.